src/actions/getMintlist.ts

```diff
@@ -14,21 +16,36 @@ const defaultIpfsGateway = "https://ipfs.io/ipfs/";
 
 export const getMintlist = async (config: Config) => {
   let mintlist: Maybe<Mintlist>;
+  let merkleTreeIndex = 0;
+
+  const airWild2Sale = await getAirWild2SaleContract(
+    new ethers.VoidSigner(""),
```

Comment:
You won't be able to read unless you give either a signer or a provider as this function requires.
If you need a provider, check out the `config` object.